### PR TITLE
settings: Set correct default value for shell's disable-user-extensio…

### PR DESCRIPTION
**Pull request by uajain**
…ns key

https://phabricator.endlessm.com/T28777

### CODE REVIEW

File: settings/dconf-defaults/settings [hunk 1]
```diff
@@ -23,4 +23,6 @@ show-in-lock-screen=false
 enabled-extensions=['ubuntu-appindicators@ubuntu.com']
 development-tools=false
 disabled-extensions=['']
-disable-user-extensions=true
+# disable-user-extensions disables all extensions keeping [enabled|disabled]-extensions
+# list unchanged.
+disable-user-extensions=false
```
Comment (wjt) [inline settings/dconf-defaults/settings]: Do we need to override this? Isn't `false` the default?

Comment (uajain) [reply settings/dconf-defaults/settings]: Well, the lockdown [documentation](https://help.gnome.org/admin/system-admin-guide/stable/dconf-lockdown.html.en) explicitly states to set the value first, in the keyfile, before locking it down(in lockfiles)

> Before you can lock down a key or subpath, you need to set it. This example shows how to lock a background setting once it has been set.

I am aware that it is defaulted to false,  but still (inclining towards the docs).

Comment (wjt) [reply settings/dconf-defaults/settings]: Why do we need to lock this down at all?

Comment (GeorgesStavracas) [reply settings/dconf-defaults/settings]: When PAYG move to be an extension, we absolutely mustn't allow users to disable extensions.

**Pull request merged**